app/src/main/res/values/strings.xml

```diff
@@ -23,5 +23,8 @@
     <string name="connection_error_message">Erreur de connexion</string>
     <string name="give_answer_here_hint">Entrez ici votre réponse</string>
     <string name="qedit_logo_description">QEDit Logo</string>
+    <string name="textview">TextView</string>
+    <string name="delete">delete</string>
```

Comment:
Anthony wants an upper case on delete...

Reply:
Lots of typos in strings.xml...